web-app/src/app/routes/alert/alert-setting/alert-setting.component.ts

```diff
@@ -710,6 +710,9 @@ export class AlertSettingComponent implements OnInit {
   onConnectModalCancel() {
     this.isConnectModalVisible = false;
   }
+  handleCancel() {
```

Comment:
suggest update the method name more understand like other, eg: onExportTypeModalCancel()

Reply:
ok